.github/workflows/update-pinned-reqs.yml

```
@@ -16,18 +16,13 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v4
       with:
-        fetch-depth: 0
+        persist-credentials: false
 
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: 3.13
-
-    - name: Install dependencies
-      run: python -m pip install --upgrade nox uv
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
```

```suggestion
      uses: astral-sh/setup-uv@v5
      with:
        python-version: 3.13
```

Then I do not think you need to use `--python 3.13` in the `uv` command on line 25.